lib/index.js

```diff
@@ -126,6 +155,7 @@ Ramen.prototype.identify = function(identify){
 
   // For tracking purposes
   ramenSettings._partner = 'segment.com';
+  ramenSettings.disable_location_watch = true;
```

Comment:
@ndhoule any word on when this will get pulled in? I got an email from @amillet89 asking me to update the index.md for Ramen's integration doc page, but I'm not sure if I should also amend that doc to include instructions on using #track, #group, and #page yet.